### ISSUE TITLE
Replies aren't encrypted

### ISSUE THREAD

**Issue by AdamCanady** [StreakYC/StreakSecureGmail#2]
People could be under the false pretense that their reply is encrypted. It would be great to have the encryption option in replies!!

**Comment by titan49er** [comment]
I agree. Users should have the option of encrypting replies to encrypted messages. Someone please fix this.
